virtool/db/utils.py

```diff
@@ -36,6 +36,27 @@ def apply_projection(document, projection):
     return {key: document[key] for key in document if projection.get(key, False)}
 
 
+async def check_missing_ids(collection, id_list, query=None):
```

Comment:
1. Use typing in function signature instead of docstring
2. Add tests as needed.